includes/definitions/avr-hd.yaml

```diff
@@ -0,0 +1,9 @@
+os: avr-hd
+text: 'avr-hd'
+type: appliance
+icon: exterity
+over:
+    - { graph: device_bits, text: Traffic }
+discovery:
+    - sysDescrId:
```

Comment:
We really should try and use sysObjectId for detection if possible. If not it's sysDescr anyway